### PR TITLE
docs - sgml for resource group sql commands

### PR DESCRIPTION
**Pull request by lisakowen**
notes:
- added new .sgml pages for create/alter/drop resource group
- updated syntax and relevant description content of create/alter role to include resource-group related info
- noticed that the .sgml for create/alter role was not consistent with the versions of these SQL commands in the xml docs; sgml docs do not include DENY* info.  not addressing in this PR.
- i built greenplum database and verified that "psql> \h CREATE RESOURCE GROUP" et al work. not sure if this work affects any tests, etc.

questions:
- copy/pasted from an existing .sgml file.  not sure what to do with the first comment lines.  modified the file name.  what about the date and user fields?
- "\h CREATE RESOURCE GROUP" displays the command syntax.  does any body know where and under what circumstances the rest of the info on the .sgml page displayed?
- do i need to update allfiles.sgml?

### CODE REVIEW

File: doc/src/sgml/ref/alter_resource_group.sgml [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+</synopsis>
+ </refsynopsisdiv>
+
+ <refsect1>
+  <title>Description</title>
+
+  <para>
+   ALTER RESOURCE GROUP changes the limits of a resource group. 
+   Only a superuser can alter a resource group.
+  </para.
```
Comment (dyozie) [inline doc/src/sgml/ref/alter_resource_group.sgml]: `</para.` -> `</para>`

Surprised that didn't cause an error somewhere...

**Comment by lisakowen** [comment]
thanks for catching that!  only the command syntax is displayed with "\h XXX".  hopefully someone can answer the second question above as to when/where the other info will be displayed.

**Comment by hlinnaka** [comment]
I think in GPDB, only the synopsis is used, for `\h`. In PostgreSQL, the rest of the SGML text would appear in the user manual, but in GPDB, I don't think it is used for anything.

**Comment by dyozie** [comment]
Ok - that was my suspicion.  Given that only the synopsis is used, I think we should probably remove the rest of the info.  Not that it will be used anywhere, but will probably lead to confusion as it will probably drift from the actual DITA xml source over time.

**Pull request merged**